build-system/tasks/lint.js

```diff
@@ -124,6 +127,19 @@ function runLinter(path, stream, options) {
       .pipe(eslint.failAfterError());
 }
 
+/**
+ * Extracts the list of JS files in this PR from the commit log.
+ *
+ * @return {!Array<string>}
+ */
+function getLintFiles() {
+  const filesInPr =
+        getStdout('git diff --name-only master...HEAD').trim().split('\n');
+  return filesInPr.filter(function(file) {
+    return path.extname(file) == '.js';
```

Comment:
In a future change, do you mind adding '.ts' to this list or making a utility method the build process uses for filtering files based on extension?

Reply:
Good suggestion. Assuming it makes sense to run `eslint` checks on `.ts` files, we should definitely include them in `gulp lint`. Will test this out via a separate PR (there are other places that need to change too, so it's kinda out of scope for this PR.)